docs/superpowers/plans/2026-08-01-devsecops-cicd-hardening.md

```
@@ -0,0 +1,533 @@
+            community.molecule/requirements.txt \
+            -o community.molecule-sbom.cdx.json || true
+      - name: Attest collection tarball provenance
+        uses: actions/attest-build-provenance@<PIN_SHA> # v2
+        with:
+          subject-path: "./community.molecule-*.tar.gz"
+      - name: Upload collection SBOM artifact
+        uses: actions/upload-artifact@<PIN_SHA> # v4
+        with:
+          name: sbom-collection
+          path: community.molecule-sbom.cdx.json
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)